src/magento-payments/payoutreports.md

```diff
@@ -0,0 +1,133 @@
+   - If you selected _Custom range_, choose the applicable date range for your payout report.
+1. View your new report in the Payout reports list.
+
+## Download Payout reports
+
+1. On the _Admin_ sidebar, go to **Sales** > **Magento Payments** > **Payout reports**.
+1. To download multiple reports, check to select them from the list and click the _Download_ (![]({% link images/images/btn-download.png %})) icon.
+1. To download a single report, click the _Download_ (![]({% link images/images/btn-download.png %})) icon in the Actions column for a specific report.
+
+Your report(s) are downloaded in a .csv format.
+
+## Available Payout reports
+
+These Payout reports are currently available in Magento Payments.
+
+Report | Description
+------------ | --------------------
+Daily | Available by default in the payments list grid for the last seven days, this report shows information for a single day.
+Today | Available from the Payout dates selector, this report shows information for the current day.
+Last 7 days | Available from the Payout dates selector, this report shows information for the last seven days.
+Last 30 days | Available from the Payout dates selector, this report shows information for the last 30 days.
+Last 90 days | Available from the Payout dates selector, this report shows information for the last 90 days.
+Year to date | Available from the Payout dates selector, this report shows information for the the entire year to date.
+Custom range | Available from the Payout dates selector, this report can be filtered to show a custom date range.
+{:style="table-layout:auto;"}
+
+## Column descriptions
+
+Payout reports include the following information.
+
+Column | Description
+------------ | --------------------
+Provider | Payment provider (PayPal)
+Provider trans | PayPal transaction ID
+Trans date | Date and time transaction was initiated
+Type | Transaction type (PAYMENT, AUTH, BONUS, CHARGEBACK, CORRECTION, CURRENCY_CONVERSATION, DEPOSIT, DISBURSEMENT, DISPUTE, FEES, HOLD, HOLD_RELEASE, INCENTIVES, OTHERS, RECOUP, REFUND, REVERSAL, WITHDRAWAL) <br> <br>See [Transaction types](#transaction-types) for more information.
+Status | Current status of the transaction (SUCCESS, DENIED, PENDING)
+Code | Transaction code that indicates either Credit or Debit
+Reference ID | Original PayPal transaction ID for which this event is related
+Invoice | Invoice ID (one per order) of the transaction
+Magento order | Magento order ID
+Magento trans | Magento transaction ID
+Pay method | Credit card type (BANK, PAYPAL, CREDIT_CARD) and associated card provider (Visa, MasterCard, etc.)
+Trans amt | Amount of the transaction
+Cur (trans amt) | Currency unit for transaction amount
+Pending | Amount yet to be disbursed
+Cur (pending) | Currency unit for the pending amount
+Seller amt | Amount of funds transferred to or from a customer; Funds moving out of the seller account show a dash (-) prefix.
```

Comment:
We might want to discuss this use of `;` in meeting. It hasn't been done in User Guide, but we can talk more about solutions for mixing non-sentence forms with sentences in tables.

Reply:
What is your suggestion in lieu of this? None of the other rows have complete sentences so this was my way of conforming to that convention while also adding the secondary info.